### DISCUSSION TITLE
[QUESTION] Clarification on Redis version support

### DISCUSSION THREAD

**Discussion by stopforumspam** [apache/kvrocks#763] [General]
I love kvrocks but it would be useful if there was a version vs Redis version support.  Whilst supported commands are listed, their Redis version isnt.

E.g. LPOP in Redis 6.2 supports a [count] parameter, allowing for multiple records to be returned whilst kvrocks returns just a single record.

I love would LPOP (etc) and all commands to be in line with Redis release but realise that this isn't going to be the case but it would be great if kvrocks published which Redis version each command is compatible with

**Comment by ShooterIT** [comment]
Thanks, we should describe that in documents, currently most commands are compatible with Redis 4.0. It is hard to support all command of Redis latest release, we will implement more commands gradually.
